lib/orbital/models/response.rb

```diff
@@ -79,6 +79,18 @@ def self.orbital_response_params(response)
           :params_tx_ref_num => extract(response, 'tx_ref_num'),
         }
       end
+
+      def first_reference_id
+        params_tx_ref_num
+      end
+
+      def second_reference_id
+        params_order_id
+      end
+
+      def gateway_error_code
+        params_resp_code
```

Comment:
Good question! I don't know if they are different or not. Will let you know if I find out something.